linkedList/app/src/main/java/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,63 @@
+package linkedlist;
+
+public class LinkedList {
+    Node head = null;
+    Node tail = null;  // not strictly required
+
+    public LinkedList(){
+        this.head = null;
+        this.tail = null;
+    }
+
+
+    //Will insert at the end of the linkedList seemed more challenging
```

Comment:
I see what you're doing here, and it's good to see people seeking out more challenges! Keep in mind that these challenges will build on each other. You'll be using your LinkedList class in other challenges, and those challenges will be written assuming that your `insert` method inserts nodes at the front, so keep this in mind for troubleshooting.